xbrowse_server/api/views.py

```diff
@@ -1100,6 +1102,18 @@ def add_individual(request):
                                                'project_id':project_id,
                                                'insertion_date':datetime.datetime.now()
                                                })
+        #update the contact information store if any updates were made
+        updated_contact_name,updated_contact_href = affected_patient['contact']['name'],affected_patient['contact']['href']
+        try:
+            seqr_project = SeqrProject.objects.get(name=project_id)
+            seqr_project.mme_primary_data_owner=updated_contact_name
+            seqr_project.mme_contact_url=updated_contact_href
+            seqr_project.save()
+        except ObjectDoesNotExist:
+            print "error: couldn't update the contact name and href of MME submission:",updated_contact_name,updated_contact_href
```

Comment:
use python 3 syntax

Reply:
for print statements use python 3 syntax (import from future)
print("error: couldn't update th...")

(use logger)

Reply:
```logger.error("ERROR: couldn't update the contact name and href of MME submission: ", updated_contact_name, updated_contact_href)```

Reply:
- spaces after comma

Reply:
Adjusted